manim/camera/camera.py

```diff
@@ -1,3 +1,4 @@
+"""A Camera converts a Scene into visual data."""
```

Comment:
Is "data" the right word here?
Maybe "A Camera converts a Mobjects of a Scene into displayed animations" instead?

Reply:
Hmm I don't like "displayed animations". We are using the verb "display" for adding a mobject or animation to the scene. What a Camera does is it takes all of that information and make a numpy array that contains RGB values per pixel. That's why I chose data. How about:

"A Camera converts the mobjects contained in a Scene into an array of pixels"